### PR TITLE
docs: Add drayman to showcase

### PR DESCRIPTION
**Pull request by jansivans**

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -1301,7 +1301,15 @@ const Users: User[] = [
     website: 'https://haochenqi.com/',
     source: 'https://github.com/HaochenQ/Haochen-Blog',
     tags: ['personal', 'opensource']
-  }
+  },
+  {
+    title: 'Drayman',
+    description: 'Server-side component framework',
+    preview: require('./showcase/drayman.png'),
```
Comment (lex111) [inline website/src/data/users.tsx:1308]: We need exactly screenshot of your site (_I mean, with navbar_), look at screenshots of other sites as an example.

Comment (jansivans) [reply website/src/data/users.tsx:1308]: Done

**Comment by slorber** [comment]
thanks 👍

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -1302,6 +1302,14 @@ const Users: User[] = [
     source: 'https://github.com/HaochenQ/Haochen-Blog',
     tags: ['personal', 'opensource']
   },
+  {
+    title: 'Drayman',
+    description: 'Server-side component framework',
+    preview: require('./showcase/drayman.png'),
+    website: 'http://drayman.io/',
```
Comment (slorber) [inline website/src/data/users.tsx:1309]:
```suggestion
    website: 'https://drayman.io/',
```

**Pull request merged**